Detectors/ITSMFT/MFT/reconstruction/src/devices/FileSink.cxx

```diff
@@ -101,7 +101,7 @@ void FileSink::Init()
   
   for ( mNObjects = 0 ; mNObjects < mBranchNames.size() ; mNObjects++ ) {
 
-    LOG(INFO) << "FileSink::Init >>>>> Creating output branch \"" << mClassNames[mNObjects] << "\" with name \"" << mBranchNames[mNObjects] << "\"";
+    LOG(INFO) << R"(FileSink::Init >>>>> Creating output branch ")" << mClassNames[mNObjects] << R"(" with name ")" << mBranchNames[mNObjects] << R"(")";
 
```

Comment:
Yeah, on the other hand they do fix potential "bad escaping" problems, so I would personally be inclined to keep it, especially since the check seems to be limited to the potentially misleading / wrong cases.